src/components/NavBar.js

```diff
@@ -1,12 +1,49 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React from "react";
```

Comment:
Not following Airbnb style, just unnecessary change

Reply:
1) No one told me that we have to use Airbnb style.
2) I dont have ESLint install.
3) Stop commenting bullshit and start coding.